os/include/tinyara/mpu.h

```diff
@@ -47,9 +47,18 @@ enum {
 #define MPU_NUM_REGIONS		1
 #endif
 
+/* Number of reserved MPU regions for a board. This is defined by the vendor.
+ * If not, default value is taken to be 0.
```

Comment:
This statement seems wrong. Default value of reserved is taken as -1, so that our own region starts from zero.